build/configs/rtl8721csm/hello/Make.defs

```diff
@@ -54,7 +54,14 @@ include ${TOPDIR}/tools/Config.mk
 ARCH_FAMILY = $(patsubst "%",%,$(CONFIG_ARCH_FAMILY))
 include ${TOPDIR}/arch/$(CONFIG_ARCH)/src/$(ARCH_FAMILY)/Toolchain.defs
 
+ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)
+LDSCRIPT = rlx8721d_img2_ns.ld
+
+EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/cmse_implib.a
+LINKLIBS = $(EXTRA_LIBS)
```

Comment:
1. Let's remove this line in all of Make.defs you modified.
   Because this line will be ignored by Makefile.unix line 197. Your library will be added by EXTRA_LIBS.
   It works well without this.

2. Do you let me know why you prefer this way? If you add this in Make.defs of arch, you don't need to do this for all Make.defs of configs.

Reply:
This method/setting was added by Nitish last time when he implement TrustZone.
So I should remove line 61, remain 60 (EXTRA_LIBS), right?

Reply:
1. Right. Remove only `LINKLIBS` line, keep the `EXTRA_LIBS` line.
2. Right. That's right location. Please refer Make.defs in s5j.
https://github.com/Samsung/TizenRT/blob/master/os/arch/arm/src/s5j/Make.defs#L200

Reply:
Your suggestion is better, I have move them to under arch/../amebad. (Same as Make.defs in s5j.)
Thank you very much